lib/server/routers/qcFlagTypes.router.js

```diff
@@ -18,6 +18,19 @@ const { rbacMiddleware } = require('../middleware/rbac.middleware.js');
 exports.qcFlagTypesRouter = {
     path: '/qcFlagTypes',
     children: [
+        {
+            path: ':id',
+            children: [
+                {
+                    method: 'get',
+                    controller: QcFlagTypesController.getQcFlagTypeByIdHandler,
+                },
```

Comment:
This endpoint should not be part of this PR

Reply:
this was refactoring, I forgot to remove other occurrence of getById route